app/src/organisms/OnDeviceDisplay/RunningProtocol/RunPausedSplash.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import {
+  Flex,
+  Btn,
+  Icon,
+  JUSTIFY_CENTER,
+  ALIGN_CENTER,
+  SPACING,
+  COLORS,
+  DIRECTION_COLUMN,
+  POSITION_ABSOLUTE,
+  TYPOGRAPHY,
+  BORDERS,
+  OVERFLOW_WRAP_BREAK_WORD,
+} from '@opentrons/components'
+import styled from 'styled-components'
+
+interface RunPausedSplashProps {
+  onClose: () => void
+  errorType?: string
+  protocolName?: string
+}
+
+export function RunPausedSplash({
+  onClose,
+  errorType,
+  protocolName,
+}: RunPausedSplashProps): JSX.Element {
+  const { t } = useTranslation('error_recovery')
+
+  let subText: string | null
+  switch (errorType) {
+    default:
+      subText = protocolName ?? null
+  }
+
+  return (
+    <Btn
+      height="100vh"
+      width="100%"
+      justifyContent={JUSTIFY_CENTER}
+      alignItems={ALIGN_CENTER}
+      position={POSITION_ABSOLUTE}
+      flexDirection={DIRECTION_COLUMN}
+      gridGap={SPACING.spacing40}
+      padding={SPACING.spacing40}
+      backgroundColor={COLORS.grey50}
+      onClick={onClose}
+    >
+      <SplashFrame>
+        <Flex gridGap={SPACING.spacing32} alignItems={ALIGN_CENTER}>
+          <Icon name={'ot-alert'} size="4.5rem" color={COLORS.white} />
+          <SplashHeader>{t('run_paused')}</SplashHeader>
+        </Flex>
+        <Flex width="49rem" justifyContent={JUSTIFY_CENTER}>
+          <SplashBody>{subText}</SplashBody>
+        </Flex>
+      </SplashFrame>
+    </Btn>
+  )
+}
+
+const SplashHeader = styled.h1`
+  font-weight: ${TYPOGRAPHY.fontWeightBold};
+  text-align: ${TYPOGRAPHY.textAlignLeft};
+  font-size: 80px;
+  line-height: 94px;
```

Comment:
I checked line-height and it says `normal` I think that would be 96px that we have in typography.ts. But not 100% sure, could you check the line-height value to a designer?
 
```suggestion
  font-size: ${TYPOGRAPY.fontSize80};
  line-height:${TYPOGRAPY.lineHeight96};
```

Reply:
Sue is still fleshing these out. I'll tweak these as she updates them, but this makes sense, thanks!